cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -564,6 +576,27 @@ private void setErrorResponse(HttpServletResponse response,
     response.getOutputStream().flush();
   }
 
+  private void setSuccessResponse(HttpServletResponse response,
+      String message,
+      int responseCode,
+      boolean json)
+      throws IOException {
+    String resp;
+    if (json) {
+      Map<String, Object> respMap = new HashMap<>();
+      respMap.put("version", JSON_VERSION);
+      respMap.put("Message", message);
+      Gson gson = new Gson();
+      resp = gson.toJson(respMap);
```

Comment:
nit: since gson is used just once you can directly call `new Gson().toJson(resoMap)`